src/Microsoft.DotNet.Darc/src/DarcLib/Models/VirtualMonoRepo/SourceManifest.cs

```diff
@@ -33,36 +33,30 @@ public void UpdateVersion(string repository, string uri, string sha, string pack
         }
         else
         {
-            Repositories.Add(new RepositoryRecord 
-            { 
-                Path = repository,
-                CommitSha = sha,
-                RemoteUri = uri,
-                PackageVersion = packageVersion
-            });
+            Repositories.Add(new RepositoryRecord(repository, sha, uri, packageVersion));
         }
     }
 
-    public void DeleteSubmodule(SubmoduleRecord record)
+    public void DeleteSubmodule(SubmoduleRecord submodule)
```

Comment:
Can you name this `RemoveSubmodule` instead of `Delete` so that it's more clear we're not deleting any files but just removing the submodule from here?